.babelrc.json

```diff
@@ -1,13 +1,19 @@
 {
   "presets": [
-    [
-      "@babel/preset-env",
-      {
-        "targets": {
-          "browsers": ["current node"]
-        },
-        "modules": "commonjs"
-      }
-    ]
-  ]
+    ["@babel/preset-env", {
+        "modules": false
+    }]
+  ],
+  "env": {
+    "cjs": {
+      "presets": [
+        ["@babel/preset-env", {
+          "modules": "commonjs"
+        }]
+      ]
+    },
+    "test": {
+      "plugins": ["babel-plugin-istanbul"]
```

Comment:
This is here to support coverage using `jest-light-runner`, which is needed to run jest outside of a [`vm`](https://nodejs.org/api/vm.html) context to [prevent segmentation faults](https://github.com/nodejs/node/issues/35889):

* https://github.com/nicolo-ribaudo/jest-light-runner#partially-supported-features
* https://github.com/nicolo-ribaudo/jest-light-runner/issues/72